andela/coding-geeks-ah-frontend
# #167484547 Users should be presented with a reset password

### What does this PR do?
- Users should be presented with a reset password
### Description of Task to be completed?
- Implement forgot password functionalities
- Implement reset password functionalities
### How should this be manually tested?
- Clone this branch
- `npm install` to install all dependencies
- add this environment variable in your .env file where you are running your backend `FRONTEND_URL = http://localhost:3000`
    N.B. you will use localhost PORT according to the port you are using for your frontend app which in this case for me it is 3000
- `npm start` to run the server for the frontend
- `npm start` to run the server for the backend
- Use this route `http://localhost:3000/forgot` to go to the login page
- N.B Provide a valid email
- Check your email and use the link sent to you to reset your password
- Click on the link and get to the reset password page

### What are the relevant pivotal tracker stories?
- [changing the reset password link to fronted url](https://www.pivotaltracker.com/story/show/169040218)

### Screenshots
<img width="1440" alt="Screen Shot 2019-10-10 at 03 55 20" src="https://user-images.githubusercontent.com/45050345/66534350-3b682680-eb16-11e9-86fe-85e4dc6bec8d.png">

<img width="1440" alt="Screen Shot 2019-10-10 at 04 29 48" src="https://user-images.githubusercontent.com/45050345/66534500-add90680-eb16-11e9-8ec0-1c0ef18846fb.png">
<img width="1440" alt="Screen Shot 2019-10-10 at 03 52 27" src="https://user-images.githubusercontent.com/45050345/66534521-b6c9d800-eb16-11e9-877c-1b71f878cd4e.png">
<img width="1440" alt="Screen Shot 2019-10-10 at 04 29 05" src="https://user-images.githubusercontent.com/45050345/66534531-c21d0380-eb16-11e9-8c7b-495a59ddb508.png">
<img width="1440" alt="Screen Shot 2019-10-10 at 03 53 01" src="https://user-images.githubusercontent.com/45050345/66534535-c6e1b780-eb16-11e9-8f50-8a9b27ecf83e.png">
